vsg/__main__.py

```diff
@@ -76,9 +113,9 @@ def validate_file_exists(sFilename, sConfigName):
         sys.exit(1)
 
 
-def read_configuration_files(commandLineArguments):
+def read_configuration_files(dStyle, commandLineArguments):
```

Comment:
dStyle can still be `None` here, right?

Reply:
It would be an empty dictionary:
```python
dStyle = {}
```

Reply:
Ok, I just wanted to make sure, that you can call this without any style, just configuration.